includes/admin/payments/actions.php

```diff
@@ -284,11 +284,11 @@ function give_update_payment_details( $data ) {
 	/**
 	 * Fires after updating edited donation.
 	 *
-	 * @since 1.0
+	 * @since 1.8.9
```

Comment:
@jaydeeprami Do not remove @since tag.
You can do like this
@since 1.0
@since 1.8.9 Changes hook name give_update_edited_donation -> give_updated_edited_donation